addons/hr/tests/test_hr_employee.py

```diff
@@ -99,20 +99,6 @@ def test_employee_from_user(self):
         self.assertEqual(employee.work_email, self.res_users_hr_officer.email)
         self.assertEqual(employee.tz, self.res_users_hr_officer.tz)
 
-    def test_employee_from_user_tz_no_reset(self):
-        _tz = 'Pacific/Apia'
-        self.res_users_hr_officer.tz = False
-        Employee = self.env['hr.employee'].with_user(self.res_users_hr_officer)
-        employee_form = Form(Employee)
-        employee_form.name = 'Raoul Grosbedon'
-        employee_form.work_email = 'raoul@example.com'
-        employee_form.tz = _tz
-        employee_form.user_id = self.res_users_hr_officer
-        employee = employee_form.save()
-        self.assertEqual(employee.name, 'Raoul Grosbedon')
-        self.assertEqual(employee.work_email, self.res_users_hr_officer.email)
-        self.assertEqual(employee.tz, _tz)
```

Comment:
Actually, let's keep that test, I updated it to check the timezone is not reset when hr manager alter it. 🙂